From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

ccompy committed:
Baseline file

ccompy committed:
updated

ccompy committed:
tweak

ccompy committed:
making a folder

ccompy committed:
image file

ccompy committed:
remove touch file

ccompy committed:
more tweaks

ccompy committed:
added image link

Lisaco88 committed:
edit pass: openshift-container-platform

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-machines-linux-openshift

Lisaco88 committed:
edit pass: openshift-get-started

Lisaco88 committed:
edit pass: openshift-origin

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-machines-linux-openshift

Lisaco88 committed:
edit pass: openshift-troubleshooting

Lisaco88 committed:
edit pass: openshift-prerequisites

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-machines-linux-openshift

Lisaco88 committed:
edit pass: openshift-prerequisites

ShawnJackson committed:
edit pass: file-server-disaster-recovery

Lisaco88 committed:
edit pass: openshift-post-deployment

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into file-server-disaster-recovery

ShawnJackson committed:
edit pass: file-server-disaster-recovery

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into file-server-disaster-recovery

ShawnJackson committed:
edit pass: file-server-disaster-recovery

ShawnJackson committed:
edit pass: file-server-disaster-recovery

yaronyg committed:
Naming consistency

Changed "storage database" to just "database" for Tables so that it's consistent with the language for the other entries.

Commit:
adding code for handling response, explanation of newtonsoft, and images for accessing keys

Lisaco88 committed:
Update openshift-container-platform.md

Lisaco88 committed:
Update openshift-get-started.md

Lisaco88 committed:
Update openshift-origin.md

Lisaco88 committed:
Update openshift-post-deployment.md

Lisaco88 committed:
Update openshift-prerequisites.md

Lisaco88 committed:
Update openshift-post-deployment.md

Lisaco88 committed:
Update openshift-prerequisites.md

Lisaco88 committed:
Update openshift-troubleshooting.md

rjagiewich committed:
edit pass: b active-directory-saas-saphana-tutorial

TomArcherMsft committed:
many edits

TomArcherMsft committed:
fixing link

TomArcherMsft committed:
cleaning  up authentication step

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
Revert "RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note"

This reverts commit b7a064116b42d574dc453def54e2b91641006a4c.

motanv committed:
Merge branch 'master' into chaos_article_update

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-saphana-tutorial

rjagiewich committed:
edit pass: b active-directory-saas-saphana-tutorial

tfitzmac committed:
expanding schema docs

rjagiewich committed:
edit pass: b active-directory-saas-saphana-tutorial

hirokib committed:
started azure function docs

hirokib committed:
Merge remote-tracking branch 'msmaster/master'

rjagiewich committed:
edit pass: b active-directory-saas-saphana-tutorial

TomArcherMsft committed:
correcting config file

TomArcherMsft committed:
acrolynx pass

hirokib committed:
using azure functions to automate sql dw compute levels

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-saphana-tutorial

rjagiewich committed:
edit pass: active-directory-saas-saphana-tutorial

Commit:
Added note about public IPs

tfitzmac committed:
revising schemas

tfitzmac committed:
fixing errors

hirokib committed:
examples for schedules

hirokib committed:
header forgot space

rjagiewich committed:
Update active-directory-saas-saphana-tutorial.md

Commit:
resolving subscription key issue

Commit:
Update.

hirokib committed:
Next steps

Commit:
Java updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

TomArcherMsft committed:
interim checkin

LizCasey committed:
Changes that correspond with the directive in Task #1077555: Update references of cores --> vCPU

Commit:
Update.

TomArcherMsft committed:
fixed placeholder text

TomArcherMsft committed:
fixed placeholder text 2

ccompy committed:
updated with template details

ccompy committed:
IP and formatting tweaks

TomArcherMsft committed:
fixed typo

ccompy committed:
format fixes

TomArcherMsft committed:
reworked the plan and apply sections

TomArcherMsft committed:
tweaked section heading for consistency

TomArcherMsft committed:
tweaked section heading for consistency 2

TomArcherMsft committed:
acrolynx

TomArcherMsft committed:
consistency issue

TomArcherMsft committed:
consistency issue with link text for terraform commands

ccompy committed:
added missing quotes

TomArcherMsft committed:
tweaking/synching intro bullets and section headings

TomArcherMsft committed:
corrected link text

LizCasey committed:
Corrected invalid link to non-existing bookmark so pull request can finish.

ccompy committed:
fixes

tfitzmac committed:
revised schemas

ccompy committed:
clarified a point

ccompy committed:
added the forced tunnel doc

tfitzmac committed:
added tables

ccompy committed:
Merge branch 'master' into forced-tunnel-doc

ShawnJackson committed:
Update file-server-disaster-recovery.md

ccompy committed:
acronlinx related fixes

TomArcherMsft committed:
splitting variables file into two files

ccompy committed:
added the title

Lisaco88 committed:
Update openshift-container-platform.md

Lisaco88 committed:
Update openshift-get-started.md

Lisaco88 committed:
Update openshift-origin.md

TomArcherMsft committed:
copy edit

LizCasey committed:
Extensive copyedit to file failing merge due to low Acrolinx score.

TomArcherMsft committed:
split instructions for working in a solo env (without a saved execution plan) and working with a saved execution plan

TomArcherMsft committed:
acrolynx pass

Lisaco88 committed:
Update openshift-post-deployment.md

Commit:
Java QS

Lisaco88 committed:
Update openshift-prerequisites.md

Lisaco88 committed:
Update openshift-troubleshooting.md

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

ccompy committed:
added clarifying sentence

hirokib committed:
azure functions

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note.

dlepow committed:
GRID 5.0 updates per Huseyin

dlepow committed:
Edits

markgalioto committed:
Changing the date around Backup vault deprecation.

LizCasey committed:
Corrected the date on the file.

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

markgalioto committed:
adding more files to the list that have been changed

ccompy committed:
Updated per review guidance

Commit:
task 656: Topic overhaul and new screenshots, add multiple query string note

spelluru committed:
New article

spelluru committed:
added next step

spelluru committed:
added example

spelluru committed:
removed space

Commit:
task 656: edits

dominicbetts committed:
Change to Microsoft IoT Central

mumian committed:
new article

mumian committed:
fix the links

mumian committed:
update

mumian committed:
update

cherylmc committed:
update

sethmanheim committed:
Article updates

sethmanheim committed:
Updating version

sethmanheim committed:
Acrolinx

ShawnJackson committed:
Update openshift-post-deployment.md

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

dlepow committed:
wording update

Commit:
Fixing redirect.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

ccompy committed:
Edited the title

sethmanheim committed:
Acrolinx

tfosmark committed:
Merge pull request #27438 from dominicbetts/iot-central-name

Change to Microsoft IoT Central

ShawnJackson committed:
Update active-directory-saas-saphana-tutorial.md

Commit:
task 656: expand overview

Commit:
Merge pull request #27447 from cherylmc/peering

update

ShawnJackson committed:
Merge pull request #26453 from Lisaco88/virtual-machines-linux-openshift

edit pass: openshift-container-platform

rwike77 committed:
added warning

ShawnJackson committed:
Merge pull request #27079 from rjagiewich/active-directory-saas-saphana-tutorial

Edit pass: Active directory saas saphana tutorial

mimig1 committed:
Corrected table storage section.

dlepow committed:
typos

sethmanheim committed:
Freshness and updating version numbers

hirokib committed:
changes

PRMerger16 committed:
Merge pull request #27453 from rwike77/warning

added warning

PRMerger15 committed:
Merge pull request #27388 from markgalioto/ab-11082017

Changing the date around Backup vault deprecation.

sethmanheim committed:
Acrolinx

v-shils committed:
Merge pull request #27249 from LizCasey/v-elicas1102

Changes that correspond with the directive in Task #1077555: Update r…

Commit:
Adding new structure

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

tfitzmac committed:
deleted tag policy

itechedit committed:
Update file-server-disaster-recovery.md

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

v-shils committed:
Merge pull request #27162 from hirokib/master

Azure functions with timer scaler

Commit:
Merge pull request #27457 from sethmanheim/sb-wk2

Freshness and updating version numbers

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

PRMerger14 committed:
Merge pull request #26820 from yaronyg/patch-1

Naming consistency

markgalioto committed:
second pass on updating the date when Backup vaults will be migrated. THis completes all articles.

mimig1 committed:
link fix

itechedit committed:
Merge pull request #26783 from ShawnJackson/file-server-disaster-recovery

edit pass: file-server-disaster-recovery

Commit:
Incorporating Dinesh feedback.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
Spelling correction.

Lisaco88 committed:
Merge pull request #27077 from tfitzmac/1106schema

expanding schema docs

markgalioto committed:
updating the backup-azure-manage-windows-server.md for acrolinx

carolinacmoravia committed:
C6850: Hardcoded EN locale in link

Hello, @v-dotren,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

PRMerger13 committed:
Merge pull request #27143 from jimdial/patch-27

Added note about public IPs

spelluru committed:
Added links to Until, If, Wait activities

Commit:
Merge pull request #27460 from markgalioto/ab-11102017

second pass on updating the date when Backup vaults will be migrated.…

Commit:
Preview to see if this looks better.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

spelluru committed:
acrolynx fixes

v-shils committed:
Merge pull request #27030 from TomArcher/edit-terraform-cloud-shell-article

Edits for Create a VM cluster with Terraform and HCL

mumian committed:
update date

spelluru committed:
some info

rmca14 committed:
Merge pull request #27446 from sethmanheim/wk-sb

Article updates

spelluru committed:
Cleanup

v-shils committed:
Merge pull request #27398 from dksimpson/dean-656

task 656: Topic overhaul and new screenshots, add multiple query stri…

sethmanheim committed:
Freshness review

GitHubber17 committed:
Merge pull request #27382 from dlepow/fix

N-Series VMs: GRID 5.0 updates per Huseyin

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

twooley committed:
updated screenshot and removed outdated sentence

Commit:
Incorporating Dinesh Feedback.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

LuisBosquez committed:
update for new sample

Commit:
Merge pull request #27466 from spelluru/pipelineactivities1110

Added links to Until, If, Wait activities

PRMerger14 committed:
Merge pull request #27474 from sethmanheim/wk-fresh

Freshness review

rmca14 committed:
Merge pull request #27439 from mumian/djadds1110

new article

GitHubber17 committed:
Merge pull request #27458 from tfitzmac/1110tags

deleted tag policy

PRMerger19 committed:
Merge pull request #27464 from carolinacmoravia/patch-456

C6850: Hardcoded EN locale in link

PRMerger17 committed:
Merge pull request #27456 from mimig1/patch-161

Corrected table storage section.

Commit:
Merge pull request #27481 from LuisBosquez/master

update for new sample

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

markgalioto committed:
updating art in backup-azure-system-state and tutorial-backup-windows-server-to-azure

GitHubber17 committed:
Merge pull request #27412 from spelluru/partitioneddata1109

new article - read/write partitioned data

markgalioto committed:
adding borders on the images

GitHubber17 committed:
Merge pull request #27479 from twooley/master

updated screenshot and removed outdated sentence

Commit:
Code snippits.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #27279 from ccompy/forced-tunnel-doc

New Forced tunnel how-to doc

GitHubber17 committed:
Merge pull request #27488 from markgalioto/ab-1110B

updating art in backup-azure-system-state and tutorial-backup-windows…

tfosmark committed:
Merge pull request #26831 from JamesEarle/emotion-api-csharp

Emotion API C# code sample improvement

v-alje committed:
Merge pull request #27499 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #27362 from motanv/chaos_article_update

Chaos article update

Lisaco88 committed:
Merge pull request #27356 from roygara/javaStorageQS

Java storage qs

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
